lib/BackgroundJobs/ClassifyImagenetJob.php

```diff
@@ -12,8 +12,6 @@
 
 class ClassifyImagenetJob extends ClassifierJob {
 	public const MODEL_NAME = 'imagenet';
-	public const BATCH_SIZE = 100; // 100 images
-	public const BATCH_SIZE_PUREJS = 25; // 25 images
```

Comment:
Just to double check with the pure js variant, could the increase of the default cause follow up issues with performance or a higher crash rate?

Reply:
Mmmh, yeah, that's a good point.

Reply:
Moved to ClassifierJob and AdminSettings. Musicnn and Movinet may be a little overwhelmed by the slightly higher defaults, but I think it should be fine.

Reply:
Alternatively, I could create a Settings service that keeps track of the defaults, that would maybe be cleaner